pkg/kubelet/client/kubelet_client_test.go

```diff
@@ -53,9 +53,9 @@ func TestMakeTransportValid(t *testing.T) {
 		EnableHttps: true,
 		TLSClientConfig: restclient.TLSClientConfig{
 			CertFile: "../../client/testdata/mycertvalid.cer",
-			// TLS Configuration, only applies if EnableHttps is true.
+			// TLS Configuration, only applies if EnableHTTPs is true.
```

Comment:
`EnableHttps` is the flag here at https://github.com/kubernetes/kubernetes/pull/58437/commits/2751ccf1bfc4e5613792b671ed9d4ec766e2ed2d#diff-00db4807093c02bb99fd7c3c4d9f5bc5L53, so need for this change I think.

Reply:
Yes, it should be `EnableHTTPS`.